docs/docs/guides/labs/components/building-pipelines-with-components/adding-components.md

```diff
@@ -8,7 +8,7 @@ import DgComponentsPreview from '@site/docs/partials/\_DgComponentsPreview.md';
 
 <DgComponentsPreview />
 
-To add components to your project, you can instantiate them from the command line, which will create a new directory inside your `components/` folder that contains a `component.yaml` file.
+To add components to your project, you can instantiate them from the command line, which will create a new directory inside your `components/` folder that contains a `defs.yaml` file.
```

Comment:
`defs/` folder

Looks like we should scrub this whole file for this sort of stuff